completions/tmatrix-completion.bash

```diff
@@ -0,0 +1,43 @@
+#
```

Comment:
No need for a full license header, just add the shortened SPDX-License-Identifier version like it is in the source files.

Reply:
Sorry, didn't pay attention to that.